pkgs/applications/science/math/wolfram-engine/notebook.nix

```diff
@@ -0,0 +1,9 @@
+{ lib, stdenv, writeScriptBin, jupyter, wolfram-for-jupyter-kernel }:
+
+let
+  wolfram-jupyter = jupyter.override { definitions = { wolfram = wolfram-for-jupyter-kernel.definition; }; };
+in
+  writeScriptBin "wolfram-notebook" ''
+    #! ${stdenv.shell}
```

Comment:
We want interactive most likely.

Reply:
I'm not sure what do you mean. The `wolfram-notebook` script launches the `Jupyter` server and open a web page in a browser where to edit notebooks.

Reply:
My bad.

Reply:
I understand you are suggesting to add the `-i` flag. Given the purpose of the script explained in my previous comment, I don't think it's necessary.

Reply:
No, bashInteractive is a bash build with readline support, nothing else.

Reply:
I see thanks for the explanation. The shell in that script doesn't have any interaction with the user. It's just used to launch a server. What's the benefit of using `bashInteractive` in this case?

Reply:
hmmm should be fine I think